services/time/tests/test_exonum_time.rs

```diff
@@ -368,7 +368,7 @@ fn test_selected_time_less_than_time_in_storage() {
     let mut testkit = TestKitBuilder::validator()
         .with_validators(1)
         .with_rust_service(TimeServiceInstance)
-        .with_simple_supervisor()
+        .with_rust_service(SimpleSupervisor)
```

Comment:
Do we still need the `.with_simple_supervisor()` method ?

Reply:
I removed it within 5a9f90e